dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -99,3 +99,11 @@ lease.renewal.threshold.desc=The percentage of the duration of the recovery log
 enable.log.retries=Enable retries on database transaction logs
 enable.log.retries.desc=When recovery logs are stored in an RDBMS table, this property allows SQL operations that fail to be retried.
 
+shutdown.on.log.failure=Shutdown server on transaction log failure
+shutdown.on.log.failure.desc=Specifies if an application server that is part of a recoveryGroup will be terminated by the transaction manager if a non-recoverable error occurs accessing its own transaction service logs.
+
+log.retry.interval=Interval between tran log retries on failure
+log.retry.interval.desc=Specifies the interval in seconds between attempts to recover from a transient error accessing the transaction service logs.
+
+log.retry.limit=Max number of tran log retries on failure
```

Comment:
Are "max" and "tran" labels/names used in the product? If not, use "Maximum" and "transaction"